config/bootstrap.php

```diff
@@ -33,3 +35,6 @@ class_alias('Cake\Mailer\Transport\SmtpTransport', 'Cake\Network\Email\SmtpTrans
 
 // Sets the initial router state so future reloads work.
 Router::reload();
+
+// Add table as model factory using the default table locator.
+ModelFactory::add('Table', [TableRegistry::locator(), 'get']);
```

Comment:
This uses the default ```TableLocator```, its still possible to override it from all classes using ```ModelAwareTrait``` like before.